tst/components/ui/Modal/ModalButtonGroup.test.tsx

```diff
@@ -12,34 +12,103 @@ import lightTheme from '../../../../src/theme/light';
 import { renderWithTheme } from '../../../test-helpers';
 
 describe('ModalButtonGroup', () => {
-  it('renders a group with a Button', () => {
-    const onClose = jest.fn();
-    const labelID = 'test-label';
-    const testContext = { onClose, labelID };
+  const primaryButtonLbl = 'test-primary';
+  const secondaryButtonLbl = 'test-secondary';
+  const onClose = jest.fn();
+  const labelID = 'test-label';
+  const testContext = { onClose, labelID };
+
+  it('renders a group with a primary Button', () => {
+    const component = (
+      <ModalContext.Provider value={testContext}>
+        <ModalButtonGroup
+          primaryButtons={[<ModalButton label={primaryButtonLbl} />]}
+        />
+      </ModalContext.Provider>
+    );
+    const { getByTestId, getByLabelText } = renderWithTheme(
+      lightTheme,
+      component
+    );
+    const el = getByTestId('modal-button-group');
+    expect(el).toBeInTheDocument();
+    expect(getByLabelText(primaryButtonLbl)).toBeInTheDocument();
+  });
+
+  it('renders a group with a primary Button & secondary Button', () => {
+    const component = (
+      <ModalContext.Provider value={testContext}>
+        <ModalButtonGroup
+          primaryButtons={[<ModalButton label={primaryButtonLbl} />]}
+          secondaryButtons={[<ModalButton label={secondaryButtonLbl} />]}
+        />
+      </ModalContext.Provider>
+    );
+    const { getByTestId, getByLabelText } = renderWithTheme(
+      lightTheme,
+      component
+    );
+    const el = getByTestId('modal-button-group');
+    expect(el).toBeInTheDocument();
```

Comment:
nit: this is asserted in the test case above, we may only assert what we aim to test in this particular case. Can remove these 2 lines.